src/openzaak/management/commands/migrate_vestigingen_to_nnps.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2025 Dimpact
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from openzaak.components.zaken.models import NietNatuurlijkPersoon, Vestiging
+
+
+class Command(BaseCommand):
+    help = "Migrate deprecated Vestiging rollen to NietNatuurlijkePersoon."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        self.stdout.write(
+            f"Attempting to migrate {Vestiging.objects.count()} Vestigingen..."
+        )
+
+        migration_count = 0
+        for vestiging in Vestiging.objects.all():
+
+            if NietNatuurlijkPersoon.objects.filter(rol=vestiging.rol).exists():
+                self.stdout.write(
+                    self.style.WARNING(
+                        f"Vestiging: {', '.join(vestiging.handelsnaam)} could not be migrated, "
+                        f"a NietNatuurlijkPersoon already exists on Rol: {vestiging.rol.uuid}"
+                    )
+                )
+                continue
+
+            nnp = NietNatuurlijkPersoon.objects.create(
+                statutaire_naam=", ".join(vestiging.handelsnaam),
```

Comment:
Not sure to be honest why `handelsnaam` was an array, but I guess we can only do a best effort to try and fit this into `statutaire_naam`